client/src/components/LogInModal.jsx

```diff
@@ -13,26 +13,22 @@ const LogInModal = ({ isOpen, onClose }) => {
     //Call useNavigate hook to change component display upon log in
     const navigate = useNavigate();
 
+    //State to hold various error messages
+    const [formErrorMessage, setFormErrorMessage] = useState(null);
+
     const handleSubmit = async (event) => {
         event.preventDefault();
-        
-        //Check that correct value is being submitted right now
-        console.log(event.currentTarget.username.value);
-
-        //Set the value of the username input field to be stored in username variable
-        const username = event.currentTarget.username.value;
+        setFormErrorMessage(null);
 
-        //Future Plans: Add in password functionality - OAuth?
-
-        //If on submission, there is a username submitted, send it to the server to check its existence
-        if (username) {
+        //If on submission, both credentials are submitted, send them to the server to check authorization
+        if (credentials.email && credentials.password) {
             try {
-                const response = await fetch('/api/users/', {
+                const response = await fetch('/api/users/login', {
                     method: 'POST',
                     headers: {
                         'Content-Type': 'application/json',
                     },
-                    body: JSON.stringify({ username })
+                    body: JSON.stringify(credentials)
```

Comment:
Are you still ending the password to the server unencrypted? If you aren't adding that now, add a comment for you to come back to and update.